tests/integration/test_fim/test_registry/test_registry_wildcards/test_registry_wildcards.py

```diff
@@ -0,0 +1,249 @@
+
+    wazuh_min_version: 4.5.0
+
+    tier: 1
+
+    parameters:
+        - configure_local_internal_options_module:
+            type: fixture
+            brief: Set the local_internal_options for the test.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_syscheckd:
+            type: fixture
+            brief: Clear the Wazuh logs file and start a new monitor.
+
+    assertions:
+        - Verify that the FIM database is in 'full database alert' mode
+          when the maximum number of values to monitor has been reached.
+        - Verify that proper FIM events are generated while the database
+          is in 'full database alert' mode.
+
+    input_description: A test case (fim_registry_limit) is contained in external YAML file (wazuh_conf.yaml)
+                       which includes configuration settings for the 'wazuh-syscheckd' daemon. That is combined
+                       with the testing registry key to be monitored defined in this module.
+
+    expected_output:
+        -
+
+    tags:
+        - scheduled
+    '''
+
+    monitored_keys = get_messages(generate_monitoring_callback(CB_FIM_WILDCARD_EXPANDING), timeout=T_10)
+    assert monitored_keys != [], f"Did not receive expected '{CB_FIM_WILDCARD_EXPANDING}' events"
+
+    subkey = monitored_keys[0].replace(f"{WINDOWS_HKEY_LOCAL_MACHINE}\\", "")
+    subkey = subkey+f"\\{key_name}"
+    path = monitored_keys[0] + f"\\{key_name}"
+
+    # Create custom key and custom value
+    reg_handle = create_registry(registry_parser[WINDOWS_HKEY_LOCAL_MACHINE], subkey, KEY_WOW64_64KEY)
+    modify_registry_value(reg_handle, value_name, REG_SZ, 'added')
+    event_added = check_registry_crud_event(callback=callback_value_event, path=path, type='added', timeout=T_10,
+                                            arch='x64')
+    assert event_added is not None, 'Did not find the expected "registry_value added" event'
+
+    # Add new value in the key and detect the modification of created monitored key is detected
+    modify_registry_value(reg_handle, value_name, REG_SZ, 'modified')
+    event_modified = check_registry_crud_event(callback=callback_value_event, path=path, type='modified', timeout=T_10,
+                                               arch='x64')
+    assert event_modified is not None, 'Did not find the expected "registry_value modified" event'
+
+    # Delete the created key and check it's deletion is detected
+    delete_registry_value(reg_handle, value_name)
+    event_deleted = check_registry_crud_event(callback=callback_value_event, path=path, type='deleted', timeout=T_10,
+                                              arch='x64')
+    assert event_deleted is not None, 'Did not find the expected "registry_value deleted" event'
+
+    # Delete key to clean envirmoent
```

Comment:
Fix typo

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3856/commits/b50ad9625566811faf8e88847b0fb2542f4bcb69